pycontrib/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 # Distributed under the original FontForge BSD 3-clause license
 
 install(FILES excepthook.py DESTINATION ${CMAKE_INSTALL_DATADIR}/fontforge/python)
+install(DIRECTORY simple DESTINATION ${CMAKE_INSTALL_DATADIR}/fontforge/python)
```

Comment:
I don't see any reason for this to be bundled

Reply:
I can only see that it was installed by the autotools build system, no idea what it is for:

https://github.com/fontforge/fontforge/blob/71cde275d083dbea87b1e8d0a6baaf7144189ea3/pycontrib/simple/Makefile.am#L29

Reply:
It's a bunch of toy scripts, but they're basic/half-broken and are not easily discoverable, so yeah, no reason to bundle it

Reply:
Ok, dropped that.